libs/datasets/tail_filter.py

```diff
@@ -132,8 +132,8 @@ def _filter_one_series(self, series_in: pd.Series) -> pd.Series:
                 ),
                 # `name` is a tuple with elements _EXPECTED_INDEX_NAMES
                 location_id=series_in.name[0],
-                variable=series_in.name[2],
-                bucket=series_in.name[1],
+                variable=series_in.name[1],
+                bucket=series_in.name[2],
```

Comment:
this might be helpful as a namedtuple

Reply:
You are not the first with that idea: https://stackoverflow.com/questions/26658240/getting-the-index-of-a-row-in-a-pandas-apply-function#comment110869633_26658301 and https://github.com/pandas-dev/pandas/issues/8574#issuecomment-59735379 says there is an issue but I can't find it.